public/app/ui/components/collections/table-results.html

```diff
@@ -1,26 +0,0 @@
-<table class="table table-bordered table-striped results-table">
```

Comment:
This file has been merged into `search-results.html` (template for `SearchResultsComponent`).